apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-snap-scroll-example.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, ChangeDetectionStrategy } from '@angular/core';
+
+@Component({
+    selector: 'fdp-dynamic-page-snap-scroll-example',
+    templateUrl: './platform-dynamic-page-snap-scroll-example.component.html',
+    styleUrls: ['./platform-dynamic-page-snap-scroll-example.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformDynamicPageSnapScrollExampleComponent {
+    onCollapseChange(event: Event): any {
```

Comment:
event any same as mentioned above, please update where ever it is repeated.

Reply:
added appropriate event type and return type as void.